lib/index.js

```diff
@@ -111,26 +111,29 @@ class ServerlessFargateTasks {
 
       let desired = options.tasks[identifier]['desired']
 
-      // create the service definition
-      var service = {
-        'Type': 'AWS::ECS::Service',
-        'Properties': Object.assign({
-          'Cluster': {"Fn::Sub": '${FargateTasksCluster}'},
-          'LaunchType': 'FARGATE',
-          'ServiceName': name,
-          'DesiredCount': desired == undefined ? 1 : desired,
-          'TaskDefinition': {"Fn::Sub": '${' + normalizedIdentifier + 'Task}'},
-          'NetworkConfiguration': {
-            'AwsvpcConfiguration': Object.assign({
-              'AssignPublicIp': options.vpc['public-ip'] || "DISABLED",
-              'SecurityGroups': options.vpc['security-groups'] || [],
-              'Subnets': options.vpc['subnets'] || [],
-            }, network_override),
-          }
-        }, service_override)
+      // check for optional service if set
+      var allowService = options.tasks[identifier]['allowService'] || true
```

Comment:
@JeeemJeerms `|| true` doesn't seem right here?  since `false || true` will always be true?

Reply:
@JeeemJeerms it is an OR statement. If the first one is false it will always be the second one:

```
> false || true
true
```

Reply:
fixed